cross/squid/Makefile

```
@@ -1,28 +1,45 @@
 PKG_NAME = squid
-PKG_VERS = 3.1.22
+PKG_VERS = 5.0.4
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://www.squid-cache.org/Versions/v3/3.1/
-PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_NAME = $(shell echo '$(PKG_NAME)' | tr '[:lower:]' '[:upper:]')_$(subst .,_,$(PKG_VERS)).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/squid-cache/squid/archive
+PKG_DIR = $(PKG_NAME)-$(shell echo '$(PKG_NAME)' | tr '[:lower:]' '[:upper:]')_$(subst .,_,$(PKG_VERS))
```

I'm not sure shelling out is worth it. Just using `SQUID` is good enough IMHO (same with PKG_DIST_NAME)
```suggestion
PKG_DIR = $(PKG_NAME)-SQUID_$(subst .,_,$(PKG_VERS))
```

---

I'm actually surprised I did that... let me rework that.

---

solved.